package.json

```diff
@@ -10,6 +10,9 @@
     "lerna:publish": "lerna publish from-git --yes",
     "ci": "npm-run-all build lint format:validate test",
     "test": "lerna run test --stream",
+    "test:java-parser": "mocha packages/java-parser/test/*.js",
```

Comment:
it is better if the upper level package.json would not include specific
instructions for scripts belonging to inner packages.

Reply:
I will remove them, I just realised that we can use scope with lerna